src/server.ts

```diff
@@ -51,7 +50,6 @@ app.get(
  */
 
 app
-  .get("/set-tracking-preferences", setTrackingPreferences)
```

Comment:
@damassi I'm aware of this and it will be dealt with.

Reply:
Are you sure that's sufficient to evade safari's clearing? According to their engineers, the only way to persist is if one sets a secure cookie on the server (i could be misunderstanding the domain point above)

Reply:
Interesting and weird. Might as well do it then I suppose.

Reply:
Interestingly, this seems to not be more widely known 🤷

Reply:
Resolved in https://github.com/artsy/force/pull/12280/commits/871452a1a15c10847a50fe08e3ea1f9cdcdfe48a